spec/controllers/hub/clients_controller_spec.rb

```diff
@@ -356,10 +356,10 @@
           expect(tobias_2018_assignee).to have_text "Lindsay"
         end
 
-        context "when a client needs a response" do
+        context "when a client is flagged" do
           before { tobias.touch(:flagged_at) }
 
-          it "adds the needs response icon into the DOM" do
+          it "adds the flagged icon into the DOM" do
```

Comment:
this is just preemptive description-changing. in a future commit i'll completely decouple the flag from any SLA-related fields